Python/bltinmodule.c

```diff
@@ -1623,15 +1623,18 @@ hash as builtin_hash
     obj: object
     /
 
-Return the hash value for the given object.
+Return the integer hash value for the given object within this process.
```

Comment:
I struggled on which way to word this.  do people, especially newbies, understand what a process is?  i've also seen linter wording around this say "between runs" but that is less technical, what even is a run?  people come at interactive python use from so many environments that I don't know if the concept of a run or process makes sense.  but process is at least technically accurate.  as hash randomization, for example, is process based.  and hashes of things like `object()` are the pointer values and thus process specific.

Reply:
I agree with @JelleZijlstra - given the addition to the next paragraph, I don't think you need this addition here.